cypress/e2e/project-list.cy.ts

```diff
@@ -22,6 +21,7 @@ describe("Project List", () => {
 
     it("renders the projects", () => {
       const languageNames = ["React", "Node.js", "Python"];
+      const statusNames = ["Critical", "Warning", "Stable"];
```

Comment:
It's good that you used the same approach as with the names. Makes it consistent and easy to read